mysite/blog/forms.py

```diff
@@ -0,0 +1,8 @@
+from django import forms
+
+
+class EmailPostForm(forms.Form):
+    name = forms.CharField(max_length=25)
+    email = forms.EmailField()
+    to = forms.EmailField()
+    comments = forms.CharField(required=False, widget=forms.Textarea)

```

Comment:
This name is ambiguous since the post will also be having comments. We can rename it as share_text.